docs/src/main/sphinx/connector/hive.rst

```diff
@@ -1663,15 +1663,8 @@ with Parquet files performed by the Hive connector.
         optimized parquet reader by default. The equivalent catalog session
         property is ``parquet_optimized_reader_enabled``.
       - ``true``
-    * - ``parquet.optimized-writer.enabled``
-      - Whether the optimized writer is used when writing Parquet files.
-        Set this property to ``false`` to disable the optimized parquet writer by
-        default. The equivalent catalog session property is
-        ``parquet_optimized_writer_enabled``.
-      - ``true``
     * - ``parquet.optimized-writer.validation-percentage``
```

Comment:
Should we also rename `parquet.optimized-writer.validation-percentage` to `parquet.writer.validation-percentage` ?

Reply:
I think that would be good.. but also keep the legacy config working as usual